<?xml version="1.0"?>

<launch>
  <arg name="x_pos" default="3.0"/>
  <arg name="y_pos" default="0.0"/>
  <arg name="z_pos" default="0.0"/>

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="paused" value="false"/>
    <arg name="gui" value="true"/>
  </include>


  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
     <arg name="name" default="chirya"/>
     <arg name="model" value="$(find intelligent_drone)/urdf/hector_camera_laser.urdf.xacro"/>
   </include>



  <group ns="tb3">
    <param name="robot_description_tb3" command="$(find xacro)/xacro --inorder $(find turtlebot3_description)/urdf/turtlebot3_burger.urdf.xacro" />
    <node pkg="gazebo_ros" type="spawn_model" name="spawn_urdf_tb3" args="-urdf -model turtlebot3_burger -x $(arg x_pos) -y $(arg y_pos) -z $(arg z_pos) -param robot_description_tb3" />
  </group>


</launch>